resources/resincli.robot

```diff
@@ -151,6 +155,17 @@ Check if "${option}" variable "${variable_name}" with value "${variable_value}"
     ${result} =  Run Process    echo "${result_env.stdout}" | grep ${variable_name} | grep " ${variable_value}"    shell=yes
     Process ${result}
 
+Check if "${option}" variable "${variable_name}" with value "${variable_value}" exists on "${type}" "${id}"
+    [Documentation]    Available values for argument ${option} are: ENV, CONFIG
+    @{list} =  Create List    ENV    CONFIG
+    Should Contain    ${list}    ${option}
+    ${result_env} =  Run Keyword If    '${option}' == 'ENV'    Run Process    balena envs --${type} ${id} | sed '/ID[[:space:]]*NAME[[:space:]]*VALUE/,$!d'    shell=yes
+    ...    ELSE
+    ...    Run Process    balena envs --config --${type} ${id} | sed '/ID[[:space:]]*NAME[[:space:]]*VALUE/,$!d'    shell=yes
+    Process ${result_env}
+    ${result} =  Run Process    echo "${result_env.stdout}" | grep ${variable_name} | grep " ${variable_value}"    shell=yes
```

Comment:
Couldn't we use the `--json` flag in the previous command to get something that can be validated with jq instead of grep, for accuracy?

Reply:
sure, if you want to add a commit to handle it - go for it.